### PR TITLE
Update OrientDB versions

### PR DESCRIPTION
**Pull request by robfrank**
2.1: updated to 2.1.16

2.2: updated to 2.2.0-beta2

2.2 serie is now based on alpine while 2.1 is kept as previous

### CODE REVIEW

**Comment by yosifkit** [comment]
Did you want to also remove `2.1.14`?

Changeset: https://github.com/orientechnologies/orientdb-docker/compare/031dae0993...a0833290305

LGTM (build test running)

File: library/orientdb [hunk 1]
```diff
@@ -4,6 +4,10 @@
 
 2.1.14: git://github.com/orientechnologies/orientdb-docker@031dae09937615329b22dc7805a5114dd3510734 2.1
```
Comment (tianon) [inline library/orientdb]: To add a little color to @yosifkit's question: removing it here will remove it from the "Supported" section on the Hub readme (and will prevent us from spending cycles rebuilding it on the official build server), but the tag will still be available to users who want it.  (See https://github.com/docker-library/official-images#library-definition-files for more detail on this.)

File: library/orientdb [hunk 2]
```diff
@@ -4,6 +4,10 @@
...
 
+2.1.16: git://github.com/orientechnologies/orientdb-docker@a083329030504cfc87f5198a28912e0e4ea5e7ec 2.1
+
 2.2.0-beta: git://github.com/orientechnologies/orientdb-docker@60ea1215c2c9c1204a8add48b18295dde8392991 2.2
```
Comment (tianon) [inline library/orientdb]: The same question probably applies to this tag, too.

**Comment by robfrank** [comment]
Hi, I removed 2.1.14 and 2.2.0-beta leaving only latests versions for each branch. I'll pay  more attention on next PR.

**Comment by yosifkit** [comment]
Thanks! LGTM, Build test of #1647; 57eb614 (`orientdb`):

```console
$ bashbrew build "orientdb"
Cloning orientdb (git://github.com/orientechnologies/orientdb-docker) ...
Processing orientdb:2.0.18 ...
Processing orientdb:2.1.16 ...
Processing orientdb:2.2.0-beta2 ...
Processing orientdb:latest ...
$ bashbrew list --uniq "$url" | xargs test/run.sh
testing orientdb:2.0.18
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...passed
	'override-cmd' [4/4]...passed
testing orientdb:2.1.16
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...passed
	'override-cmd' [4/4]...passed
testing orientdb:2.2.0-beta2
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...passed
	'override-cmd' [4/4]...passed
```

**Comment by tianon** [comment]
LGTM

**Pull request merged**